packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,663 @@
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+          iconRight={<IconArrowRight />}
+        >
+          {state.activeStep === 5 ? 'Send' : 'Next'}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Overflow = (args) => {
+  const reducer = commonReducer(12);
+
+  const initialState = {
+    activeStep: 1,
+    states: [
+      'available',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+      'disabled',
+    ],
+  };
+  const labels = [
+    'Step 1 - longer text',
+    'Step 2',
+    'Step 3',
+    'Step 4',
+    'Step 5',
+    'Step 6',
+    'Step 7',
+    'Step 8',
+    'Step 9',
+    'Step 10',
+    'Step 11',
+    'Step 12',
+  ];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <div style={{ maxWidth: '400px' }}>
+      <Stepper
+        labels={labels}
+        language="en"
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={12}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+      <div
+        style={{
+          height: '300px',
+          display: 'flex',
+          justifyContent: 'flex-start',
+          alignItems: 'flex-end',
+          gap: '24px',
+          marginLeft: '10px',
+        }}
+      >
+        <Button
+          disabled={state.activeStep === 1}
+          variant="secondary"
+          onClick={() => dispatch({ type: 'setActive', payload: state.activeStep - 1 })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+          iconLeft={<IconArrowLeft />}
+        >
+          Previous
+        </Button>
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+          iconRight={<IconArrowRight />}
+        >
+          {state.activeStep === 12 ? 'Send' : 'Next'}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const WithCustomTheme = (args) => {
+  const reducer = commonReducer(5);
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled', 'disabled'],
+  };
+
+  const labels = ['Step 1 - longer text', 'Step 2', 'Step 3', 'Step 4', 'Step 5'];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const theme = {
+    '--hds-stepper-color': 'var(--color-tram)',
+    '--hds-step-content-color': 'var(--color-tram)',
+  };
```

Comment:
Could the example use more custom variables? So we would know they work

Reply:
It might be a good idea to test them. We originally discussed this with @ronijaakkola, and decided to go with a more simple example, but thought about the possibility to make a one that utilises more variables